src/main/minio.js

```diff
@@ -2013,6 +2013,32 @@ export class Client {
   }
 }
 
+// Promisify various public-facing APIs on the Client module.
+Client.prototype.makeBucket = promisify(Client.prototype.makeBucket)
```

Comment:
If we'd like something less verbose, I can list the functions as strings and turn this into a `for` loop.